extensions/custom_pages/elements/views/calendars.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+use humhub\modules\calendar\extensions\custom_pages\elements\CalendarEventsElement;
+use humhub\modules\calendar\widgets\CalendarEntryPicker;
+use humhub\modules\space\widgets\SpacePickerField;
+use humhub\modules\topic\widgets\TopicPicker;
+use humhub\modules\ui\form\widgets\ActiveForm;
+use humhub\modules\user\widgets\UserPickerField;
+
+/* @var CalendarEventsElement $model */
+/* @var ActiveForm $form */
+?>
+<div class="records-content-form-fields" data-type="static">
+    <?= $form->field($model, 'static')->widget(CalendarEntryPicker::class) ?>
+</div>
+
+<div class="records-content-form-fields" data-type="options">
+    <?= $form->field($model, 'space')->widget(SpacePickerField::class) ?>
```

Comment:
I would expect these fields (space, topic, author) + limit in the Base Custom Pages modules. And only the `getContentFilterOptions` in the Calendar module.

e.g. If we add a new generic/base filter later on - it would cool that it's applied to all modules automatically.

Reply:
@luke- Done in the commits https://github.com/humhub/custom-pages/commit/80b7a7286208ec5b03f54c0f3290509033694655 and https://github.com/humhub/calendar/pull/537/commits/8c0da22cfc6f990b041739464c9c7461996323a9.